src/instr/instr_paje_types.cpp

```diff
@@ -45,33 +45,14 @@ static type_t newType (const char *typeNameBuff, const char *key, const char *co
   return ret;
 }
 
-void PJ_type_free (type_t type)
+s_type::~s_type()
 {
-  value* val;
-  char *value_name;
-  xbt_dict_cursor_t cursor = nullptr;
-  xbt_dict_foreach (type->values, cursor, value_name, val) {
```

Comment:
Just to be clear: I would like the content of this->values to be cleaned up. For that, you have to restore the ```xbt_dict_foreach (type->values, cursor, value_name, val) {``` part, and use ```delete value``` in it.